webpack.config.js

```diff
@@ -252,7 +253,14 @@ module.exports = (env, argv) => {
         chunks: ['admin'],
         filename: 'admin.html',
         publicPath: '.',
-        template: 'src/admin.html'
+        template: 'cms/src/admin.html'
+      }),
+      new HtmlWebpackPlugin({
+        ...baseHtmlPluginConfig,
+        chunks: ['editor'],
+        filename: 'editor.html',
```

Comment:
It'd be better to call this `cms-editor` or something like that. That would differentiate it better from the `doc-editor`.